09_json/mohitnag/pkg/puppy/store/testdata/duplicate-puppies.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "puppyid": 1,
+        "breed": "dog",
+        "colour": "white",
+        "value": "2"
+    },
+    {
+        "puppyid": 1
+    }
+]
```

Comment:
nit: there is no end-of-line at the end of this file. All lines should be terminated by a newline, including the last line. See if you can configure your editor to always do that.

Reply:
Done